template-values-production.json

```diff
@@ -1,6 +1,7 @@
 {
   "repository": {
-    "name": "dev-sanity-gatsby-blog"
+    "name": "dev-sanity-gatsby-blog",
+    "title": "Blog (Gatsby.js + Sanity.io + Netlify)"
```

Comment:
There's no field called `title` in the repository object from Jumpstart. Only `name`, `owner` and `private`.